### PR TITLE
fix: Update resource pages and helper functions for 10.x name/id changes

### LINKED ISSUE #2947: tables app fails to load in ssr  catalogue
**What version of EMX2 are you using (see footer)**
10.x

**Describe the bug**
view does not load 
**To Reproduce**
1. Go to ['https://emx2.dev.molgenis.org/catalogue-demo/ssr-catalogue/tables](https://emx2.dev.molgenis.org/catalogue-demo/ssr-catalogue/tables)

`Field 'refSchema' in type 'MolgenisColumnType' is undefined\`

**Expected behavior**
list of tables

### PR DESCRIPTION
**Pull request by connoratrug**
Use id where possible and remove name.toLower workarounds

Closes #2947

### CODE REVIEW

File: apps/nuxt3-ssr/interfaces/types.ts [hunk 1]
```diff
@@ -254,6 +254,7 @@ export interface IColumn {
 }
 
 export interface ITableMetaData {
+  name: string;
```
Comment (mswertz) [inline apps/nuxt3-ssr/interfaces/types.ts:257]: do you still want to use that for your application? I would expect you want to use only 'id' and 'label'
(nb. label defaults to 'name' unless seperate labels are defined)

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/tables.vue [hunk 2]
```diff
@@ -29,7 +29,7 @@ const tables = computed(() =>
             <TableRow>
               <TableCell>
                 <NuxtLink :to="`/${schemaId}/ssr-catalogue/${table.id}`">{{
-                  table.id
+                  table.name
```
Comment (mswertz) [inline apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/tables.vue:32]: table.label?

**Review by mswertz — CHANGES_REQUESTED** [review]
I think you want to use table.label and column.label instead of table.name and column.name?

File: apps/nuxt3-ssr/interfaces/types.ts (hunk 1 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/interfaces/types.ts]: done

File: apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/tables.vue (hunk 2 above)
Comment (connoratrug) [reply apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/tables.vue]: done

**Review by mswertz — APPROVED** [review]

**Pull request merged**